### PR TITLE
15 draft blog post

### PR DESCRIPTION
**Pull request by Robinlovelace**
- Move post
- Intro + title

Please review @michaeldorman, @nowosad, @anitagraser

### CODE REVIEW

File: blog.qmd [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
```
Comment (Nowosad) [inline blog.qmd:1]: @Robinlovelace I assume this is for the geocompx blog?

File: blog.qmd [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+# Introduction
+
+In this blog post, we talk about our experience teaching R and Python for geocomputation, broadly defined as [follows](https://r.geocompx.org/intro#what-is-geocomputation):
+
+> Working with geographic data in a computational way, focusing on code, reproducibility and modularity.
+
+There are are two major motivations for this post:
+
+- To share our experience teaching R and Python for geocomputation, at the OpenGeoHub Summer School in Poznan, Poland
```
Comment (Nowosad) [inline blog.qmd:14]: you can write "Poznań" (with ń) if you want

File: blog.qmd [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+- To report progress on our open source book, [Geocomputation with Python](https://py.geocompx.org/)
+
+The post is based on the following lecture notes, which we recommend checking out for deeper dives into the R and Python implementations of geocomputation:
+
+- [Tidy geographic data with sf, dplyr, ggplot2, geos and friends](https://ogh23.robinlovelace.net/tidy)
+- [Working with Spatial Data in Python](https://geobgu.xyz/presentations/p_2023_ogh/)
```
Comment (Nowosad) [inline blog.qmd:20]: "Working with Spatial Data" -> "Working with spatial data" to keep the style inline with the two other workshops

File: blog.qmd [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+- [Tools and packages to query and process Sentinel-1 and Sentinel-2 data with R and Python](https://github.com/loreabad6/ogh23)
+
+# Working with vector Geographic Data
```
Comment (Nowosad) [inline blog.qmd:23]: "geographic data" (the same as in the intro)

**Review by Nowosad — COMMENTED** [review]
Just a few style comments. Let me know when the draft is more complete then I will review it once more.

File: blog.qmd (hunk 1 above)
Comment (Robinlovelace) [reply blog.qmd:1]: Good plan.

File: blog.qmd (hunk 2 above)
Comment (Nowosad) [reply blog.qmd:19]: Cool! https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request

File: blog.qmd [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+title: "Teaching R and Python for geographic data: shared experiences and lessons learned"
+# Render on save: true
+---
+
+```{r}
+#| include: false
+knitr::opts_chunk$set(message = FALSE, warning = FALSE)
+```
+
+# Introduction
```
[the remark this answers is not in the document]
Comment (Robinlovelace) [reply blog.qmd:11]: :tada:

File: blog.qmd
Comment (anitagraser) [inline blog.qmd:1]: If you want python code more similar to the r logic, there would be https://stackoverflow.com/questions/22291565/pandas-text-matching-like-sqls-like#22291631

File: blog.qmd [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+
+The following commands select a subset of the data based on attribute values.
+
+:::: columns
+
+::: {.column width="50%"}
+
+```{python}
+sel = 'Port Lotniczy Poznań-Ławica im. Henryka Wieniawskiego'
+pol = pol_all[pol_all['name'] == sel]
```
Comment (michaeldorman) [inline blog.qmd:175]: pol = pol_all[pol_all['name'].str.contains('Port*.+Poz', na=False)]

**Comment by michaeldorman** [comment]
Great start! Will be happy to help, I'm still trying to figure out how to collaborate on GitHub...

**Comment by Robinlovelace** [comment]
Thanks all I'll merge now to break CI 😱 any ideas to fix welcome and to keep momentum

File: blog.qmd
Comment (Robinlovelace) [reply blog.qmd:1]: Awesome

File: blog.qmd (hunk 6 above)
Comment (Robinlovelace) [reply blog.qmd:175]: Please put in PR to main of you get a chance

Comment (michaeldorman) [reply blog.qmd:175]: I'm trying to do it... Will appreciate any help
Screenshot from 2023-08-29 14-59-13

**Pull request merged**